src/tpm2_wrap.c

```diff
@@ -26,6 +26,11 @@
 
 /* For some struct to buffer conversions */
 #include <wolftpm/tpm2_packet.h>
+#ifndef WOLFTPM2_NO_WOLFCRYPT
+/* Required in wolfTPM2_CreateAuth() for name computation of NV handles */
+#include <wolfssl/wolfcrypt/hash.h>
+#endif
```

Comment:
I believe this section is no longer required because its in tpm2.c now.

Reply:
Indeed. Good catch